### PR TITLE
Optimization: Local span layer issue

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Issue Url: https://github.com/apache/incubator-skywalking/issues/2200
Local span layer and layer text is Unknown  => 'Local'
Snapshot like this:
image

### CODE REVIEW

File: src/components/TraceTree/d3-trace.js [hunk 1]
```diff
@@ -222,7 +222,12 @@ export default class TraceMap {
       .attr('text-anchor', 'start')
       .attr('fill', d => this.type[d.data.layer])
       .attr('stroke', d => this.type[d.data.layer])
-      .text(d => d.data.layer);
+      .text(d => {
+        if(d.data.type == 'Local' && d.data.layer == 'Unknown'){
```
Comment (zhaoyuguang) [inline src/components/TraceTree/d3-trace.js]: Is that better?

Comment (wu-sheng) [reply src/components/TraceTree/d3-trace.js]: I think so.

Comment (TinyAllen) [reply src/components/TraceTree/d3-trace.js]: I think only checking the ''Unkown" is enough, using ternary statement to do this well.

Comment (wu-sheng) [reply src/components/TraceTree/d3-trace.js]: We can't just check `unknown`, because some RPC plugin may forget to set layer.

**Comment by wu-sheng** [comment]
I am merging this. Please submit UI sync pr.

**Pull request merged**